src/Compiler/CompilerDefines.h

```diff
@@ -108,6 +108,7 @@ static constexpr const char *PLACEMENT_TIMING_LOG{"post_place_timing.rpt"};
 static constexpr const char *ROUTING_LOG{"routing.rpt"};
 static constexpr const char *ROUTING_TIMING_LOG{"post_route_timing.rpt"};
 static constexpr const char *TIMING_ANALYSIS_LOG{"timing_analysis.rpt"};
+static constexpr const char *TA_TIMING_LOG{"ta_timing_analysis.rpt"};
```

Comment:
Assuming TA is tatum, why do we have a separate report file for this ? We will either have tatum or sta so it should go into same rpt file.

Reply:
Actually TA doesn't stand for Tatum :) it's just there in order not to name the file timing_analysis_timing_analysis.rpt
will try to come up with another name